charts/akeyless-api-gateway/templates/_helpers.tpl

```diff
@@ -111,6 +111,14 @@ Generate chart secret name
 {{/*
 Check customer fragment
 */}}
+
+{{- define "akeyless-api-gw.root.config.path" -}}
+{{- if .Values.akeylessStrictMode }}
+     {{- printf "/home/akeyless" -}}
```

Comment:
@renanaAkeyless please approve this path

Reply:
I've talked with @barucoh and we saw that the code first chack the user home dir and if it can't find the files there it will check `/root`

Reply:
If the username is always `akeyless` than this should be fine, but I would use:
```
printf "$HOME"
```
I think this should always work btw, also for `root`...